unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/LoadingHUD/Bridge/LoadingBridge.cs

```diff
@@ -20,7 +19,8 @@ public void SetLoadingScreen(string jsonMessage)
 
         if (!string.IsNullOrEmpty(payload.message))
             DataStore.i.HUDs.loadingHUD.message.Set(payload.message);
-        DataStore.i.HUDs.loadingHUD.showWalletPrompt.Set(payload.showWalletPrompt);
+        // mendez: showWalletPrompt no longer needs to exist
+        DataStore.i.HUDs.loadingHUD.showWalletPrompt.Set(false);
```

Comment:
We should completly remove the wallet prompt.

I'm creating a ticket for this so we dont block this PR.
https://app.zenhub.com/workspaces/unity-6047bad476c3c0001942ee91/issues/decentraland/unity-renderer/851

Reply:
Thanks